### ISSUE TITLE
Aggregate forms > "virtual form" or "multiple custom forms for a help topic"

### ISSUE THREAD

**Issue by jeffchulg** [osTicket/osTicket#230]
Hi, I'm facing a problem in parametrization : it's not a bug, but it's quite ennoying.

```
when I create a custom form "database creation", I set a list of form fields as "RDBMS", "Version", "Server", etc.
```

If the help topic is just "Create DB on existing host", it's OK.
But if the help topic is "Create DB on new server", I'll need to create a new custom form (exact copy of the first one) and add additional informations needed for this, which could be actually the custom form for help topic "New Server".

So, basically, if there were a mecanism which allows to create a custom form with multiple parents OR add multiple forms to a help topic, it would be very good for that purpose.

**Comment by FireEmerald** [comment]
I'm facing the same problem and it's ridicules to create the same form two or three times only with some additional fields... @greezybacon Was there any change since 2013 ?

It could look like:
![example](https://cloud.githubusercontent.com/assets/1107939/10577127/c2286e4a-7668-11e5-9d8f-3a706f5ede00.jpg)

or just a little (+) next to the first drop down menu, wheres the (?) already.

**Comment by greezybacon** [comment]
This is implemented in v1.10:

![image](https://cloud.githubusercontent.com/assets/672074/10608798/35eb0bce-7706-11e5-9c0c-a4f2227aa6e4.png)

Furthermore, individual fields can be disabled from the attached forms, and the order of the forms can be organized in the topic management page.
